src/fnv1a.ts

```diff
@@ -1,5 +1,6 @@
 import { Buffer } from 'buffer';
 import { Long } from './long';
+import type { BufferEncoding } from './parser/utils';
```

Comment:
This doesn't look used, was it included erroneously? If so, shouldn't we have a lint rule disallowing imports which are never used?

Reply:
It is used in both functions in this file..? 😅 are we seeing something different 
I removed this type anyway.

Reply:
yeah I dunno, the GitHub diff was just adding this and no other changes :shrug: